src/styles/blocks/now-on-view.scss

```diff
@@ -0,0 +1,26 @@
+.now-on-view {
+  &__title {
+    margin-bottom: 6px;
+  }
+
+  &__subtitle {
+    margin-bottom: 48px;
+
+    text-align: center;
+  }
+
+  &__exhibition {
+    margin-bottom: 48px;
+    &:last-child {
+      margin-bottom: 40px;
+    }
+  }
+
+  &__link-button {
+    margin: 0 auto 76px;
+
+    @include onTablet {
```

Comment:
It's recommended to use more descriptive mixins for media queries, like 'on-tablet-and-up' or 'on-tablet-and-down' instead of just 'onTablet', to make it clear whether it's a min-width or max-width media query.